components/vocabularies/orphanet/api/src/main/resources/orphanet/conf/schema.xml

```diff
@@ -0,0 +1,286 @@
+        <charFilter class="solr.PatternReplaceCharFilterFactory" pattern="^(.*)$" replacement="SOLR_START_MARKER $1 SOLR_END_MARKER"/>
+        <tokenizer class="solr.ClassicTokenizerFactory"/>
+        <filter class="solr.KeywordMarkerFilterFactory" protected="protwords.txt"/>
+        <filter class="solr.LowerCaseFilterFactory"/>
+      </analyzer>
+    </fieldtype>
+
+    <fieldtype name="prefix_match" class="solr.TextField" positionIncrementGap="1000">
+      <analyzer type="index">
+        <charFilter class="solr.PatternReplaceCharFilterFactory" pattern="^(.*)$" replacement="SOLR_START_MARKER $1"/>
+        <tokenizer class="solr.ClassicTokenizerFactory"/>
+        <filter class="solr.KeywordMarkerFilterFactory" protected="protwords.txt"/>
+        <filter class="solr.LowerCaseFilterFactory"/>
+      </analyzer>
+      <analyzer type="query">
+        <charFilter class="solr.PatternReplaceCharFilterFactory" pattern="^(.*)$" replacement="SOLR_START_MARKER $1*"/>
+        <tokenizer class="solr.ClassicTokenizerFactory"/>
+        <filter class="solr.KeywordMarkerFilterFactory" protected="protwords.txt"/>
+        <filter class="solr.LowerCaseFilterFactory"/>
+      </analyzer>
+    </fieldtype>
+
+    <!-- Since fields of this type are by default not stored or indexed,
+         any data added to them will be ignored outright.  -->
+    <fieldtype name="ignored" stored="false" indexed="false" multiValued="true" class="solr.StrField" />
+  </types>
+
+  <fields>
+    <field name="version" type="string" indexed="true" stored="true" omitNorms="true"/>
+
+    <field name="id" type="string" indexed="true" stored="true" required="true" />
+    <field name="idStub" type="id_prefix" indexed="true" stored="false" omitNorms="true"/>
+
+    <!-- The primary name of the disease -->
+    <field name="name" type="text_general" indexed="true" stored="true"/>
+    <field name="nameSpell" type="text_spell" indexed="true" stored="false"/>
+    <field name="nameExact" type="exact_match" indexed="true" stored="false" omitNorms="true"/>
+    <field name="namePrefix" type="prefix_match" indexed="true" stored="false" omitNorms="true"/>
+    <field name="nameStub" type="text_prefix" indexed="true" stored="false" omitNorms="true"/>
+    <field name="nameSort" type="string" indexed="true" stored="false"/>
+
+    <field name="term_group" type="string" indexed="true" stored="true"/>
+
+    <field name="is_a" type="string" indexed="true" stored="true" multiValued="true"/>
+    <field name="term_category" type="string" indexed="true" stored="true" multiValued="true"/>
+    <field name="part_of" type="string" indexed="true" stored="true" multiValued="true"/>
+
+    <field name="age_of_onset" type="string" indexed="true" stored="true" multiValued="true"/>
+
+    <field name="present_in" type="string" indexed="true" stored="true" multiValued="true"/>
+
+    <field name="birth_prevalence" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
All prevalence info should be processed and stored as actual numbers, in addition to the string representations. E.g. `1 - 9 / 100,000` -> 50,000.